src/UI/controllers/HTMLTooltipUIController.js

```diff
@@ -39,6 +39,12 @@ export class HTMLTooltipUIController extends UIController {
         window.addEventListener('pointerdown', this, true)
     }
 
+    destroy () {
```

Comment:
Can you add a comment to explain why this 'destroy' method was added? I assume it's related to needing to remove the `pointerdown` event, but it's not clear why :)